libs/ui/src/admin/ArtApprovalModal/ArtApprovalForm.tsx

```diff
@@ -0,0 +1,221 @@
+import { FC, memo, useState } from 'react'
+
+import {
+  Avatar,
+  Button,
+  ButtonGroup,
+  HStack,
+  Text,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalOverlay,
+  Stack,
+  Textarea,
+  Box,
+  VStack,
+  Image,
+  Center,
+  Flex,
+} from '@chakra-ui/react'
+import { Splide, SplideSlide } from '@splidejs/react-splide'
+import {
+  HiDotsVertical,
+  HiOutlineCheck,
+  HiOutlineX,
+  HiPencil,
+} from 'react-icons/hi'
+
+import { ArtImageProps, ArtApprovalFormTypes } from './types'
+
+const ArtImage: FC<ArtImageProps> = memo(({ image, alt }) => {
+  return (
+    <Image
+      objectFit="contain"
+      src={`${process.env['NX_API_URL']}${image.url}`}
+      alt={alt}
+    />
+  )
+})
+
+export const ArtApprovalForm: FC<ArtApprovalFormTypes> = ({
+  onReject,
+  onApprove,
+  onDelete,
+  artId,
+  artDescription,
+  artTitle,
+  artImages,
+  editorId,
+  isOpen,
+  onClose,
+  // editorAvatar,
+  // editorName,
+  artArtistName,
+  isEdit,
+}) => {
+  const [feedback, setFeedback] = useState('')
+
+  const handleReject = (artId: number, editorId: number, feedback: string) => {
+    onReject(artId, editorId, feedback)
+  }
+
+  const handleApprove = (artId: number, editorId: number, feedback: string) => {
+    onApprove(artId, editorId, feedback)
+  }
+  const handleDelete = (artId: number) => {
+    onDelete(artId)
+  }
+  return (
+    <Box>
+      <Modal onClose={onClose} size={['1088px']} isOpen={isOpen}>
+        <ModalOverlay />
+        <ModalContent>
+          <ModalCloseButton />
+          <ModalBody>
+            <Stack direction={['column', 'row']} spacing={4} m={4}>
+              <Stack w={'full'}>

```

Comment:
If we have a large description, the image will be smaller I think. Could you test it?

```suggestion
            <Stack direction={['column', 'row']} spacing={4}>
              <Stack w={'full'} minW="50%">
```

You can also use SimpleGrid

```suggestion
            <SimpleGrid columns={{base: 1, lg: 2}} spacing={4}>
              <Stack>
```